nmostesting/TestHelper.py

```diff
@@ -264,6 +268,48 @@ def loader(uri):
     return schema
 
 
+def read_RSA_private_key(priavte_key_files):
+    for private_key_file in priavte_key_files:
+        private_key = open(private_key_file, "r").read()
+        if private_key.find("BEGIN RSA PRIVATE KEY") != -1:
+            return private_key
+    return None
+
+
+def generate_jwk(rsa_private_key):
+    rsa_key = RSA.importKey(rsa_private_key)
+    public_key = rsa_key.publickey().exportKey(format="PEM")
+    return JsonWebKey.import_key(public_key, {"kty": "RSA", "use": "sig",
+                                 "key_ops": "verify", "alg": "RS512"}).as_dict()
+
+
+def generate_token(rsa_private_key, scopes=None, write=False, azp=False, add_claims=True, overrides=None):
+    if scopes is None:
+        scopes = []
+    header = {"typ": "JWT", "alg": "RS512"}
+    payload = {"iss": "{}".format("https://testsuite.nmos.tv"),
+               "sub": "testsuite@nmos.tv",
+               "aud": ["https://*.{}".format(CONFIG.DNS_DOMAIN), "https://*.local"],
+               "exp": int(time.time() + 3600),
+               "iat": int(time.time()),
+               "scope": " ".join(scopes)}
+    if azp:
+        payload["azp"] = str(uuid.uuid4())
+    else:
+        payload["client_id"] = str(uuid.uuid4())
+    nmos_claims = {}
+    if add_claims:
+        for api in scopes:
+            nmos_claims["x-nmos-{}".format(api)] = {"read": ["*"]}
+            if write:
+                nmos_claims["x-nmos-{}".format(api)]["write"] = ["*"]
+    payload.update(nmos_claims)
+    if overrides:
+        payload.update(overrides)
+    token = jwt.encode(header, payload, rsa_private_key).decode()
+    return token
+

```

Comment:
These 3 functions are only used by Auth.py - would they be better moved into Auth.py?  Or perhaps create an `IS10Utils` module for these?

Reply:
Done, moved in `IS10Utils.py`